pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -540,46 +548,56 @@ For example, given the module declaration
+of module-level definitions. Phase 0 is the execution-time phase.
+
+A top-level @racket[require]
 @tech{instantiates} a module at @tech{phase} 0, if the module is not
 already @tech{instantiate}d at phase 0.  A top-level
-@racket[(require (for-syntax ....))] @tech{instantiates} a module at
+@racket[(require (for-syntax ...))] @tech{instantiates} a module at
 @tech{phase} 1 (if it is not already @tech{instantiate}d at that
-level); @racket[for-syntax] also has a different binding
+phase); @racket[for-syntax] also has a different binding
 effect on further program parsing, as described in
 @secref["intro-binding"].
 
-Within a module, some definitions are shifted by a phase already; the
+Within a module, some definitions are already shifted by a phase: the
 @racket[begin-for-syntax] form is similar to @racket[begin], but it
-shifts expressions and definitions by a relative @tech{phase} 1. 
+shifts expressions and definitions by a relative @tech{phase} +1. 
+Likewise, the @racket[define-for-syntax] form is similar to @racket[define],
+but shifts the definition by +1.
 Thus, if the module is @tech{instantiate}d at phase 1,
 the variables defined with @racket[begin-for-syntax] are created at phase 2,
 and so on. Moreover, this relative phase acts as another layer of
-prefixing, so that a @racket[define] of @racket[x] and a 
-@racket[begin-for-syntax]-wrapped
-@racket[define] of @racket[x] can co-exist in a module
+prefixing, so that @racket[x] defined with @racket[define] and
+@racket[x] defined with @racket[begin-for-syntax] can co-exist in a module
 without colliding. A @racket[begin-for-syntax] form can be nested
-within a @racket[begin-for-syntax] form, in which case definitions and
-expressions are in relative @tech{phase} 2, and so on. Higher phases are 
-mainly related to program parsing, instead of normal evaluation.
+within a @racket[begin-for-syntax] form, in which case the inner definitions and
+expressions are in relative @tech{phase} +2, and so on. Higher phases are 
+mainly related to program parsing instead of normal evaluation.
 
 If a module @tech{instantiate}d at @tech{phase} @math{n}
 @racket[require]s another module, then the @racket[require]d module is
 first @tech{instantiate}d at phase @math{n}, and so on
 transitively. (Module @racket[require]s cannot form cycles.) If a
 module @tech{instantiate}d at phase @math{n} @racket[require]s
-@racket[for-syntax] another module, the other module becomes
+another module @racket[for-syntax], the other module becomes
 @deftech{available} at @tech{phase} @math{n+1}, and it may later be
 @tech{instantiate}d at @tech{phase} @math{n+1}.  If a module that is
-@tech{available} at phase @math{n} for @math{n>0} @racket[require]s
-@racket[for-template] another module, the other module becomes
+@tech{available} at phase @math{n} (for @math{n>0}) @racket[require]s
+another module @racket[for-template], the other module becomes
```

Comment:
same suggestion as above, I think these require-specs mixed with English would be clearer as grammatically correct require-specs

Reply:
Okay.